yum.sh

```diff
@@ -37,3 +37,15 @@ if [[ -n "${upgrades}" ]] ; then
 else
   echo 'yum_upgrades_pending{origin=""} 0'
 fi
+
+# if yum-utils is not installed the skip rendering this metric 
+if [[ -f '/bin/needs-restarting' ]] ; then
```

Comment:
This ought to be `-x` instead of `-f`, since we need the file to exist _and_ be executable.